test/rdoc/test_rdoc_rubygems_hook.rb

```diff
@@ -2,10 +2,17 @@
 require 'rubygems'
 require 'fileutils'
 require 'tmpdir'
-require_relative '../../lib/rdoc/rubygems_hook'
 require 'test/unit'
 
-class TestRDocRubygemsHook < Test::Unit::TestCase
+# This test requires lib/rubygems_plugin.rb .
+# To execute this test under a ruby-core, lib/rubygems_plugin.rb should be synced to ruby/ruby.
+# But I don't do so and skip this test under the situation because of avoiding ambiguity about rubygems_plugins on ruby/ruby.
+begin
+  require_relative '../../lib/rubygems_plugin'
+rescue LoadError
+end
```

Comment:
Can we revert this change?
Because we have `RDoc::RubyGemsHook` in `rdoc/rubygems_hook.rb` again.

Reply:
Thanks! You're right. I revert this changes.